distributed/shuffle/tests/test_shuffle.py

```diff
@@ -1067,6 +1061,21 @@ def __init__(self, value: int) -> None:
         # ),
     }
 
+    if parse(pa.__version__) >= parse("12.0.0"):
+        columns.update(
+            {
+                # Extension types
+                f"col{next(counter)}": pd.array(
+                    [pd.Period("2022-01-01", freq="D") + i for i in range(100)],
```

Comment:
Non blocking, I think you can use period_range and interval_range instead, which is better here